site/_data/templates.yaml

```diff
@@ -437,8 +440,35 @@ This variable may be helpful if you wish to communicate to the user which links
   version: 1.28.0
   variables:
     - name: devicePendingIdPLink
-      type: PendingIdPLink
+      type: Object
       description: The PendingIdpLink that is generated when using the device OAuth flow.
+    - name: devicePendingIdPLink.displayName
+      type: String
+      description: A human readable name for this link to help you identify this link. This value will generally be an email address, or username.
+    - name: devicePendingIdPLink.email
+      type: String
+      description: The email address of the FusionAuth user being linked to the identity provider.
+    - name: devicePendingIdPLink.identityProviderLinks
+      type: List<Object>
+      description: "See full documentation for this object link:/docs/v1/tech/apis/identity-providers/links/#response-3[here]"
+    - name: devicePendingIdPLink.identityProviderName
+      type: String
+      description: The name of the identity provider that is being linked to.
+    - name: devicePendingIdPLink.identityProviderType
+      type: String
+      description: The type of identity provider that is being linked to.
+    - name: devicePendingIdPLink.identityProviderUserId
+      type: String
+      description: The Id for the User that is provided by the identity provider.
+    - name: devicePendingIdPLink.isLinkLimitExceeded
+      type: Boolean
+      description: A boolean that indicates if a user has exceeded the maximum number of links allowed.
+    - name: devicePendingIdPLink.user
+      type: User
+      description: The FusionAuth user that is linked to the identity provider.
+    - name: devicePendingIdPLink.username
+      type: String
+      description: The username of the FusionAuth user being linked to the identity provider.
```

Comment:
@robotdan @mooreds 
I believe the only thing left on this PR is documenting this `PendingIdPLink` object (including elaborating on the description per Daniel's comment above).  

Its fairly lengthy and I believe it's used in 4 other places on this page.  So I'm trying to decide on these options on how to handle the complex/deeper objects:
1. Copy/paste where applicable 😒  👎 
2. Is there way to "templatize" a template? Similar to using includes with liquid syntax? 😒  meh... still lots of duplication
3. Add another section at the top or within common variables for special objects? 🤔  - best option I can think of so far.

Reply:
re #2, untested with jekyll, but you could try this syntax

```
bill-to: &id001
    given  : Chris
    family : Dumars
ship-to: *id001
```

https://stackoverflow.com/questions/2063616/how-to-reference-a-yaml-setting-from-elsewhere-in-the-same-yaml-file

I'd have to play around with it, but love the idea of reusing the same text.

Reply:
Thanks, I tried using this but was having trouble merging array's of hashes.  I think its do-able but didn't want to spend a lot more time figuring it out. I just duplicated the field where applicable for now.